Unknown macro with nested macro use modules

I tried this code (minimized from core, macro there is called `int_impl`):

```rust
#[macro_use]
mod num {
    #[macro_use]
    mod macros {
        macro_rules! a {
            () => ()
        }

    }

    a!();
}

fn main() -> i32 { 0 }
```

I expected to see this happen: Compiles

Instead, this happened: Error

```text
tmp/num/mod.rs:4:1: error: unknown macro: [a]
    4 | a!();
      | ^
```

### Meta

- What version of Rust GCC were you using, git sha if possible.
875cd5ee9c6f037c7f5ace00b75631a1c45970a5

--- a/gcc/rust/resolve/rust-early-name-resolver.cc
+++ b/gcc/rust/resolve/rust-early-name-resolver.cc
@@ -63,8 +63,6 @@ EarlyNameResolver::accumulate_escaped_macros (AST::Module &module)
 	    continue;
 	  }
 
-	item->accept_vis (*this);
-
 	if (item->get_ast_kind () == AST::Kind::MACRO_RULES_DEFINITION)
 	  escaped_macros.emplace_back (item->clone_item ());
       }

--- a/gcc/testsuite/rust/compile/nested_macro_use3.rs
+++ b/gcc/testsuite/rust/compile/nested_macro_use3.rs
@@ -1,0 +1,16 @@
+#[macro_use]
+mod num {
+    #[macro_use]
+    mod macros {
+        macro_rules! a {
+            () => ()
+        }
+    }
+
+    a!();
+}
+
+fn main() -> i32 { 
+    a!();
+    0
+}

